sdk/core/Azure.Core/src/Shared/OperationInternalOfT.cs

```diff
@@ -91,7 +91,8 @@ internal class OperationInternal<T> : OperationInternalBase
         /// parameter <paramref name="operation"/>.
         /// </param>
         /// <param name="scopeAttributes">The attributes to use during diagnostic scope creation.</param>
-        /// <param name="fallbackStrategy">The fallback delay strategy when Retry-After header is not present.  When it is present, the longer of the two delays will be used. Default is <see cref="ConstantDelayStrategy"/>.</param>
+        /// <param name="fallbackStrategy">The fallback delay strategy when Retry-After header is not present.  When it is present, the longer of the two delays will be used.
+        /// Default is <see cref="FixedDelayStrategy"/>.</param>
```

Comment:
Should we add an overload `WaitForCompletion(DelayStrategy delayStrategy, CancellationToken cancellationToken)` which allows users to pass in their stategy in an LRO scenario?